app/src/main/res/layout/fragment_add_drink.xml

```diff
@@ -75,6 +75,7 @@
         android:layout_height="wrap_content"
         android:src="@drawable/ic_check_white_24dp"
         android:layout_margin="24dp"
+        android:backgroundTint="@color/colorPrimary"
```

Comment:
Refactor, integrate primary color in floating button (or change default accent color to match ?)